llvm/test/CodeGen/AMDGPU/function-args-inreg.ll

```diff
@@ -0,0 +1,359 @@
+; RUN: llc -march=amdgcn -mcpu=gfx900 -mattr=-flat-for-global -verify-machineinstrs < %s | FileCheck -enable-var-scope -check-prefixes=CIGFX89,GFX89 %s
```

Comment:
Remove -mattr=-flat-for-global, the only real reason to use that was trying to share manually written tests ages ago

Reply:
done.